juriscraper/scotus/scotus_email.py

```diff
@@ -30,24 +30,27 @@ def data(self) -> dict:
         """Extract all the data in the email into the following format:
 
         {
-            "filing_name": "Amicus brief of Corey J. Biazzo, Esq. submitted.",
-            "case_name": "Donald J. Trump, President of the United States, et al., Petitioners v. V.O.S. Selections, Inc., et al.",
-            "case_link": "https://www.supremecourt.gov/search.aspx?filename=/docket/DocketFiles/html/Public/25-250.html",
             "docket_number": "25-250",
-            "notification_datetime": "2025-10-09 02:04:05+00:00"
+            "notification_datetime": "2025-10-09 02:04:05+00:00",
+            "case_name": "Donald J. Trump, President of the United States v. V.O.S. Selections, Inc.",
+            "case_url": "https://www.supremecourt.gov/search.aspx?filename=/docket/DocketFiles/html/Public/25-250.html",
+            "entry_description": "Amicus brief of Corey J. Biazzo, Esq. submitted."
         }
 
-        :return: `dict` of data extracted from the email. A field will be set
-        to `None` if extraction of that specific field from the email failed or
-        if there was an error while parsing the email.
+        :return: `dict` of data extracted from the email. Will return `{}` if
+        email parsing has not occurred or has failed. A field will be set to
+        `None` if extraction of that specific field from the email fails.
         """
-        return {
-            "docket_number": self._parse_docket_number(),
-            "notification_datetime": self._parse_datetime(),
-            "case_name": self._parse_case_name(),
-            "case_url": self._parse_case_link(),
-            "entry_description": self._parse_filing_name(),
-        }
+        if self.is_valid:
+            return {
+                "docket_number": self._parse_docket_number(),
+                "notification_datetime": self._parse_datetime(),
+                "case_name": self._parse_case_name(),
+                "case_url": self._parse_case_link(),
+                "entry_description": self._parse_filing_name(),
+            }
+        else:
+            return {}
```

Comment:
Just a small suggestion: we can remove this `else` and unindent the `return {}`.